Jenkinsfile.release

```diff
@@ -31,10 +39,13 @@ ansiColor('xterm') {
           sshagent (credentials: ['0f7ec9c9-99b2-4797-9ed5-625572d5931d']) {
             // add the ssh host key for the package uploader step
             // These environment variables are defined in the "Properties Content" section for the job.
-            sh """echo "${PKG_SSH_HOST} ${PKG_SSH_HOSTKEY_TYPE} ${PKG_SSH_HOSTKEY}" | sudo -E tee -a sudo -E tee -a /etc/ssh/ssh_known_hosts"""
+            sh """echo "${PKG_SSH_HOST} ${PKG_SSH_HOSTKEY_TYPE} ${PKG_SSH_HOSTKEY}" | sudo -E tee -a /etc/ssh/ssh_known_hosts"""
 
             withDockerRegistry([credentialsId: 'docker-hub-credentials']) {
-              sh """sudo -E ci/pipeline release $params.version $params.gitsha $params.latest"""
+              sh """if [ \$(md5sum /usr/local/bin/amm | awk '{print \$1}') = '07143cdf89e94b0a957f5656193d88d7' ]; then sudo curl -L -o /usr/local/bin/amm https://github.com/lihaoyi/Ammonite/releases/download/1.0.3/2.12-1.0.3 && sudo chmod +x /usr/local/bin/amm; fi"""
+
+              if (params.dryRun == false)
+                sh """sudo -E env MARATHON_VERSION=$params.version ci/pipeline release --checkout $params.ref --run-tests=$params.runTests docker-tag linux-packages jar-artifact s3-package """ + (params.latest ? "docker-latest" : "")
```

Comment:
Why is the version passed as an environment parameter? Also, what is the reason for not triggering `linux-packages`, `jar-artifact` and `s3-package` with `release`?

Reply:
Splitting up the various published phases into various targets allows one to more easily rerun a part of the job without needing to be worried about affecting other already published artifacts. It enables more rapid iteration to fix something if it is broken.

We hardcode, in the Jenkins file, that all targets should be run (minus, of course, docker-latest).